config/scripts/hwcheck.sh

```diff
@@ -14,7 +25,7 @@ function setbuffersizes()
   echo "Setting buffersizes temporary. Consider doing this permanently"
   sudo sysctl -w net.core.rmem_max=$rmemsize
   sudo sysctl -w net.core.wmem_max=$rmemsize
-  sudo sysctl -w net.core.netdev.max_backlog=$backlogsize
+  sudo sysctl -w $BACKLOG=$backlogsize
```

Comment:
We could also write to a sysctl.conf file if the backlog size is off to set up persistent changes, but I'm not sure if it's worth it since other "prerequisites" already exist for the network (check and set MTU for used network interfaces and switch(es); which we could handle only in part with similar edits to ifcfg-files). As you prefer.